.github/workflows/CODE_REVIEW_REPORT.yml

```diff
@@ -0,0 +1,19 @@
+name: CODE REVIEW REPORT
+on:
+  #pull_request:
```

Comment:
<h3>$\color{#15F4EE}Miscellaneous$</h3>

Uncomment the pull_request trigger if it is intended to be used, or remove it if not.